### ISSUE TITLE
Error when using commonService with Spring Boot 2.4.2

### ISSUE THREAD

**Issue by gclayburg** [gclayburg/dockerPreparePlugin#7]
When using this dockerprepare config:

```groovy
dockerprepare {
    commonService = ['org.springframework.boot:spring-boot-starter-web']
}
```

and spring boot 2.4.2, I get this error:
```
$ ./gradlew clean build
> Task :expandBootJar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':expandBootJar'.
> Warning: Could not find file /home/gclaybur/dev/gvsync/upbannerdemo/build/docker/dependenciesLayer2/BOOT-INF/lib/spring-boot-starter-web-2.4.2.jar to copy.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.7.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 5s
8 actionable tasks: 8 executed
```

**Comment by gclayburg** [comment]
Indeed, the BOOT-INF/lib/spring-boot-starter-web-2.4.2.jar  file does not exist during a build.
This is failing when the plugin assumes that there is a jar file to move, based on the dependency graph.  Apparently, more recent Spring Boot versions no longer produce a jar dependency.

A workaround is to just remove the commonService setting

**Comment by gclayburg** [comment]
fixed in v1.4.1
